README.md

```diff
@@ -17,15 +17,28 @@ Lint JS, JSX, TypeScript, and more inside Markdown.
 
 ### Installing
 
-Install the plugin alongside ESLint v6 or greater:
+Install the plugin alongside ESLint v8 or greater:
```

Comment:
Which versions of ESLint will be officially supported? Per this sentence and the CI update, it looks like only >=8. However, there are still instructions for v6 and v7 below, and package.json still has `"peerDependencies": { "eslint": "^6.0.0 || ^7.0.0 || ^8.0.0"  }` .

Reply:
I'm going to update the supported ESLint versions in another PR.